alf/algorithms/ppg_algorithm.py

```diff
@@ -145,6 +149,18 @@ def __init__(self,
     def _trainable_attributes_to_ignore(self):
         return ['_aux_algorithm']
 
+    @staticmethod
+    def _validate_data_transformer(data_transformer):
+        """Returns True if UntransformedTimeStep is present and is applied
+        before all other data transformers.
+
+        """
+        if type(data_transformer) is UntransformedTimeStep:
+            return True
+        if type(data_transformer) is SequentialDataTransformer:
```

Comment:
It's always wrapped in SequentialDataTransformer?

Reply:
Verified and fixed. Thanks!